tasks/lib/helpers/git.js

```diff
@@ -52,7 +52,7 @@ function checkout(grunt, dir, branchName, done) {
 
 function merge(grunt, dir, mergeFromBranch, msgPrefix, msg, done) {
 
-  exec('git merge --no-ff -m "' + msgPrefix + ' ' + msg + '" ' + mergeFromBranch, {
+  exec('git merge --no-ff --strategy-option theirs -m "' + msgPrefix + ' ' + msg + '" ' + mergeFromBranch, {
```

Comment:
Its the same as "resolve merge conflict by using theirs". So since we merge our temp branch into develop/master it will always override the changes on develop/master with the ones from our temp branch, which is fine, since it came from master/develop couple of seconds ago :)